{
  "status": "error",
  "message": "generator didn't stop after throw()"
}